src/ui/common/src/components/pages/artifact/id/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { faCircleCheck } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Button, CircularProgress } from '@mui/material';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+import {
+    ArtifactResult,
+    handleGetArtifactResults,
+} from '../../../../reducers/workflow';
+import { AppDispatch, RootState } from '../../../../stores/store';
+import UserProfile from '../../../../utils/auth';
+import DefaultLayout from '../../../layouts/default';
+import KeyValueTable from '../../../tables/KeyValueTable';
+import StickyHeaderTable from '../../../tables/StickyHeaderTable';
+import { LayoutProps } from '../../types';
+
+type ArtifactDetailsHeaderProps = {
+    artifactName: string;
+    createdAt?: string;
+    sourceLocation?: string;
+};
+
+const ArtifactDetailsHeader: React.FC<ArtifactDetailsHeaderProps> = ({
+    artifactName,
+    // TODO: add these back once we have support for getting createdAt and sourceLocation.
```

Comment:
All we really have here is the name of the artifact result.

Don't really even have a status here either, so I'm just showing the check icon for now.

Happy to remove the check icon if we find that the status indicator here is meaningless for an artifact result.